.github/workflows/tests.jsonnet

```diff
@@ -572,6 +572,11 @@ local ignore_md = {
       ],
     } + ignore_md,
   },
+  // These extra permissions are needed by some of the jobs, e.g. build-test-javascript.
```

Comment:
you can use gha.write_permissions here, and if you understand those fields, please add comments
in gha.libsonnet